examples/kmeans/kmeans.py

```diff
@@ -0,0 +1,136 @@
+import argparse
+import contextlib
+import matplotlib
```

Comment:
`matplotlib`, `six` and `numpy` are third party libraries. Move them after system libraries.